elements/research/ddiotune.hh

```diff
@@ -0,0 +1,127 @@
+#ifndef CLICK_DDIOTune_HH
+#define CLICK_DDIOTune_HH
+#include <click/element.hh>
+#include <rte_pci.h>
+extern "C"{
+#include<pci/pci.h>
+}
+
+CLICK_DECLS
+
+/*
+=title DDIOTune
+
+=c 
+
+DDIOTune(N_WAYS, DEV [,PRINT])
+
+=s test
+
+Tune the DDIO capacity based on the input number of ways.
+
+=d 
+
+Configure IIO_LLC_WAYS register in Intel processors. In addition, this element can disable DDIO for a specific NIC.
```

Comment:
It would be great to add a link to more information (not yet available I guess) and like 3 lines about what should be the parameters in which situations. And also 2 lines of context about DDIO and ways.

FYI, this element doc will automatically go to the wiki once I generate the doc (from time to time).

Reply:
I will add more info ;)